mod-di-converter-storage-server/src/main/resources/templates/db_scripts/associations-migration/clean_profile_wrappers.sql

```diff
@@ -41,6 +41,9 @@ BEGIN
     EXECUTE 'UPDATE ' || r.table_name || ' SET masterwrapperid=null, detailwrapperid=null;';
   END LOOP;
 
+  RAISE NOTICE 'Removing new associations.';
+  TRUNCATE associations CASCADE;
```

Comment:
did you change the name of this table to `profile_associations`?

Reply:
yes, good catch! will fix it, thanks!